src/app/hero-detail/hero-detail.component.css

```diff
@@ -0,0 +1,25 @@
+    /* HeroDetailComponent's private CSS styles */
+    label {
+        color: #435960;
+        font-weight: bold;
+    }
+    input {
+        font-size: 1em;
+        padding: .5rem;
+    }
+    button {
+        margin-top: 20px;
+        margin-right: .5rem;
+        background-color: #eee;
+        padding: 1rem;
+        border-radius: 4px;
+        font-size: 1rem;
+    }
+    button:hover {
```

Comment:
The hover colour of these buttons in particular has an inaccessible colour contrast of 1.44

One way to see this in the chrome browser is if you bring up the browser dev tools and force the hover state on the button:

<img width="831" alt="Screenshot 2023-02-01 at 04 58 22" src="https://user-images.githubusercontent.com/5038459/215893239-60c804ed-fa49-46d0-b359-6bae7a687a8c.png">

Then click on the `color` of the button:

<img width="355" alt="Screenshot 2023-02-01 at 04 58 46" src="https://user-images.githubusercontent.com/5038459/215893485-bbab6585-e9d1-4b9a-9641-f88faf2f94bc.png">

You can play around with the colours right in the browser to see which combo of background + foreground (`color`) works.

Related reading: https://www.a11yproject.com/posts/what-is-color-contrast